### PR TITLE
[BE-256]: update rmv2

### PR DESCRIPTION
**Pull request by tmavroeid**

### CODE REVIEW

File: docs/project/reward-mechanism-v2.mdx [hunk 1]
```diff
@@ -5,74 +5,232 @@ sidebar_label: Reward Mechanism v2
...
 
-:::note For example:
-If the capacity of a cell is 5, but has 6 stations with a random *reward_score*, then only the first 5 devices with the highest *reward_score* will be rewarded. If there are two stations in the 5th place, with exactly the same *station_reward_score*, then the more senior of the two will get the rewards
-:::
+  \
+* **Quality-of-Data (QoD)**: An algorithm that evaluates the quality of weather data provided by a weather station. It generates a score that denotes the confidence about the quality of the weather data for each weather station. WeatherXM values weather stations that produce meaningful and usable data and the QoD score is an attempt to quantify this metric. The end goal is to encourage weather station owners to do their best to comply with WeatherXM's guidelines in order to consistently achieve the best QoD score possible.
+
+  \
```
Comment (PavlosTze) [inline docs/project/reward-mechanism-v2.mdx:49]: What are these used for @tmavroeid ? I think these `\` on that list are unnecessary.

File: docs/project/reward-mechanism-v2.mdx [hunk 2]
```diff
@@ -5,74 +5,232 @@ sidebar_label: Reward Mechanism v2
...
+* **Hardware-Class (HC)**: Each type of approved hardware (weather station) belongs to a specific Hardware Class. Hardware Classes are used to award more tokens to weather stations that have special capabilities, increased sensor accuracy or range, new sensor types, or other capabilities that make them more valuable to the network. For the time being all Hardware Classes are considered equal.
+
+  \
+* **Cell Capacity (CC)**: Each cell has a pre-defined **cell capacity** that depends on its geospatial characteristics. The cell's **capacity** is the maximum number of stations that may receive rewards in a specific cell. If `N` weather stations are active in a cell with a capacity `X` during a specific day, with `N > X`, only the first `N` stations (ordered first by **reward_score**, then by seniority) will receive rewards
+
+To read more please see our [whitepaper](https://weatherxm.com/whitepaper)
```
Comment (PavlosTze) [inline docs/project/reward-mechanism-v2.mdx:55]: A dot is missing at the end.

**Review by PavlosTze — CHANGES_REQUESTED** [review]
Some very minor changes @tmavroeid and we are good to go.

**Review by NikosTsiligaridis — APPROVED** [review]
Gj @tmavroeid. 
I will be adding more details on QoD, PoL, CC, etc and consolidate the RM1 + RM1.5 + RM2 into a single doc soon, to help clarify our current status and future direction, so I will move stuff around in this article.

File: docs/project/reward-mechanism-v2.mdx (hunk 2 above)
Comment (NikosTsiligaridis) [reply docs/project/reward-mechanism-v2.mdx:52]: ninja vision.

**Pull request merged**